src/main/groovy/org/shipkit/internal/gradle/VersionUpgrade.java

```diff
@@ -0,0 +1,84 @@
+package org.shipkit.internal.gradle;
+
+import java.io.File;
+
+/**
+ * Configuration of {@link VersionUpgradeConsumerPlugin}
```

Comment:
I think you didn't finish the second comment? About the first one - there is already a package like that, I will move all the classes there.